test/Java8andUp/src/org/openj9/test/annotationClassLoader/Test_ClassLoader.java

```diff
@@ -70,4 +76,83 @@ protected Class loadClass(String className, boolean resolve) throws ClassNotFoun
 		}
 	}
 
+	@Test
+	public void test_latestUserDefinedLoader() {
+		CachedRowSet crs = null;
+		SyncProvider syncProvider = null;
+		String customSyncProviderClassName = CustomSyncProvider.class.getName();
+		
+		try {
+			crs = (CachedRowSet)Class.forName("com.sun.rowset.CachedRowSetImpl").newInstance();
+			crs.setSyncProvider(customSyncProviderClassName);
+			syncProvider = crs.getSyncProvider();
+		} catch (Exception e) {
```

Comment:
Just let the testcase throw.  The test runner will automatically mark a test with an unexpected exception as failed

Reply:
fixed.